src/components/Survey.js

```diff
@@ -1,65 +1,110 @@
-import { NONE } from 'apisauce';
+
 import React, { useState } from 'react'
-import { SafeAreaView, StyleSheet, ScrollView, View, Text, Button, FlatList, TouchableOpacity, TextInput,Alert } from 'react-native';
-import {Field, reduxForm } from 'redux-form'
+import { SafeAreaView, StyleSheet, ScrollView, View, Text,TouchableOpacity ,Alert } from 'react-native';
+import {Field, reduxForm, destroy } from 'redux-form'
+import { connect, useDispatch } from 'react-redux';
+import {saveUser}  from '../Redux/Actions/operationActions'
+import { useNavigation } from '@react-navigation/native';
+import axios from 'axios'
+import RenderField from '../components/RenderField'
```

Comment:
Give meaningful names to your components. 

`RednerField` is really generic. In this case you are using a `TextInput`, so you can name this `TextInputField` for example

Reply:
I finished renaming my RednerField component to TextInputField